docs/content/docs/try-flink-ml/java/build-your-own-project.md

```diff
@@ -28,7 +28,7 @@ under the License.
 
 This document provides a quick introduction to using Flink ML. Readers of this
 document will be guided to create a simple Flink job that trains a Machine
-Learning Model and use it to provide prediction service.
+Learning Model and uses it to provide prediction service.
 
 ## Maven Setup
```

Comment:
Should we also have a Prerequisites section similar to the python ``build-you-own-project.md`?